tests/cpptest.sh

```
@@ -181,6 +181,11 @@ test_run() {
   try_run 'Comment after include' 73 "#include \"tmp.h\" /*block\n*/ // line\nint main(){return FOO;}"
   pp_error 'Token after include comment' "#include \"tmp.h\" /*block\n*/ illegal-token\nint main(){return FOO;}"
 
+  mkdir test_include
```

Would you add `-p` option to suppress error on second time, please?